### PR TITLE
created add-misc-node-property job

### PR DESCRIPTION
**Pull request by zhangy42**
remade upsert to node collection based on new requirement.

https://www.pivotaltracker.com/story/show/121049511

@benbp @pscharla 
@RackHD/corecommitters

### CODE REVIEW

**Comment by rackhdmirror** [comment]
@RackHD/corecommitters please review.

File: lib/jobs/add-misc-node-property-job.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+// Copyright 2016, EMC, Inc.
+/* jshint node:true */
+
+'use strict';
+
+module.exports = AddMiscNodePropertyFactroy;
+
+var di = require('di');
+
+di.annotate(AddMiscNodePropertyFactroy, new di.Provide('Job.Add.Misc.Node.Properties'));
```
Comment (benbp) [inline lib/jobs/add-misc-node-property-job.js]: I think it's probably better named like `Job.AddNodeMiscProperties`, the periods usually denote some sort of hierarchical classification.

**Comment by yyscamper** [comment]
@zhangy42 : Why not define the task & corresponding task schema?

**Comment by benbp** [comment]
@brianparry see https://github.com/RackHD/on-core/pull/211#issuecomment-248426715

**Comment by PengTian0** [comment]
test this please

**Comment by zhangy42** [comment]
It took too long to discuss the necessity of this feature and I am closing this PR as I will anyway rebase the code before we merge the code.

**Pull request closed without merging**